localisation/00_wc_vanilla_replacements.csv

```diff
@@ -260,3 +260,7 @@ rival_skull_desc;As one stares into the empty sockets of this skull, it is not u
 obj_create_treasury_desc;[This.GetTitledFirstName] wants to gather artifacts to put in [This.GetHerHis] empty treasury.;[This.Get_Le_TitledFirstName] veut r�unir des artefacts pour son Tr�sor vide.;[This.GetTitledFirstName] m�chte Artefakte sammeln und sie in der leeren Schatzkammer ausstellen.;;[This.GetTitledFirstName] quiere reunir artefactos que poner en su c�mara del tesoro vac�a.;;;;;;;;;x
 MODIFIER_ARMY_REINFORCE_RATE;Commanded Army Reinforcement Rate;;Armee-Verst�rkungsrate;;;;;;;;;;;x
 MODIFIER_CHURCH_OPINION;Church Opinion;;Kirchenmeinung;;;;;;;;;;;x
+hermetics_write_magnum_opus;Write Down Whispers;;Gefl�ster niederschreiben.;;;;;;;;;;;x
+hermetics_write_magnum_opus_desc;Write down the whispers of the Old Gods that constantly ring in your ears.;;Schreibt das Fl�stern der alten G�tter auf, das Euch st�ndig in den Ohren klingt.;;;;;;;;;;;x
+hermetics_write_magnum_opus_power_desc;Enables �Gright-clicking�! on your own portrait in order to write down the mysterious whispers of your masters in a book.\nQuality of the book depends on amount of your �Y[This.TrueReligion.GetPietyName]�!.;;Erm�glicht �GRechtsklick�! auf das eigene Portr�t, um die geheimnisvollen Einfl�sterungen der Meister in einem Buch niederzuschreiben.\nDie Qualit�t des Buches h�ngt von der Menge Eures �Y[This.TrueReligion.GetPietyName]�! ab.;;;;;;;;;;;x
```

Comment:
Since you added this part in power description instead, `\n` is no longer needed.
```suggestion
hermetics_write_magnum_opus_power_desc;Enables �Gright-clicking�! on your own portrait in order to write down the mysterious whispers of your masters in a book. Quality of the book depends on amount of your �Y[This.TrueReligion.GetPietyName]�!.;;Erm�glicht �GRechtsklick�! auf das eigene Portr�t, um die geheimnisvollen Einfl�sterungen der Meister in einem Buch niederzuschreiben. Die Qualit�t des Buches h�ngt von der Menge Eures �Y[This.TrueReligion.GetPietyName]�! ab.;;;;;;;;;;;x
```